docs/quickstart/expressions.mdx

```diff
@@ -67,7 +67,7 @@ Actions are referenced by a sluggified version of their name.
   Check out the [workflow triggers](/tutorials/workflow-triggers.mdx) tutorial for a detailed guide setting up webhooks for workflows.
 </Info>
 
-Workflows can be triggered via webhook, manual UI trigger, or the `Execute Child Workflow` action.
```

Comment:
was renamed to execute subflow

Reply:
Going to deal with renaming everything to subflows in another PR